evm/src/executor/inspector/cheatcodes/util.rs

```diff
@@ -71,6 +72,24 @@ pub fn apply<DB: Database>(
             state.labels.insert(inner.0, inner.1.clone());
             Ok(Bytes::new())
         }
+        HEVMCalls::ToString0(inner) => {
+            Ok(ethers::abi::encode(&[Token::String(inner.0.pretty())]).into())
+        }
+        HEVMCalls::ToString2(inner) => {
+            Ok(ethers::abi::encode(&[Token::String(inner.0.pretty())]).into())
+        }
+        HEVMCalls::ToString3(inner) => {
+            Ok(ethers::abi::encode(&[Token::String(inner.0.pretty())]).into())
+        }
+        HEVMCalls::ToString4(inner) => {
+            Ok(ethers::abi::encode(&[Token::String(inner.0.pretty())]).into())
+        }
+        HEVMCalls::ToString5(inner) => {
+            Ok(ethers::abi::encode(&[Token::String(inner.0.pretty())]).into())
+        }
+        HEVMCalls::ToString6(inner) => {
+            Ok(ethers::abi::encode(&[Token::String(inner.0.pretty())]).into())
+        }
```

Comment:
this is indeed weird, need to debug this in `abigen!`

Reply:
using expand, it indeed produces 0 to 6, skipping 1.